### ISSUE TITLE
Feature request: Short cuts

### ISSUE THREAD

**Issue by dekuenstle** [v1r0x/polls#32]
Pressing keys to fast select green/yellow/red on the field you hover would be nice at first.
If you could vote completely with the keyboard it would be awesome.

**Comment by v1r0x** [comment]
Don't know what would be a good system (e.g. user friendly). Any ideas?

**Comment by dekuenstle** [comment]
I'm also not sure but have a suggestion: 
I think allways using three keys of the homerow (no breaking fingers) for the three states of  a field would help.
Then the question occurs, for which field the keypress should account.
If the mouse hovers over a field of the user this should be 'glowing'. Clicking on it should toggle through the states like in the status quo. But pressing on of the keys (like s d f) should jump directly to the state.
Another possible improvement would be, if you could change the selected field with the arrow keys. So at first its the current hovered and can be changed with left/right arrow, but if mouse moves it will select again the current on.

**Comment by freddie-freeloader** [comment]
dudle does have the following binding afaik:
* TAB - `next`
* UP - Set `okay`
* DOWN - Set `not okay`

**Comment by dartcafe** [comment]
Many work to test in every release. In times of mobile deviced, I would like to concentrate on click and touch. Close this? I think, we will not find someone to develop and test this every time.

**Comment by v1r0x** [comment]
I'd keep this open and implement it. Shouldn't be that hard to test and as long as the layout/IDs in the DOM don't change, we only have to test it once.
News uses shortcuts as well and I use them a lot.

**Comment by dartcafe** [comment]
@v1r0x Still a planned feature?

**Comment by v1r0x** [comment]
Yes, definitely! Do you want it as part of 1.0?

**Comment by dartcafe** [comment]
No. Not for 1.0 and I still have concerns regarding this feature.

**Comment by stefan123t** [comment]
The shortcuts that freddie-freeloader mentioned: Tab (next option), Up (vote yes) and Down (vote no), Shift-Tab (previous option) are also relevant for accessibility in general, see #1354.
I would suggest adding question mark ? (vote Maybe) and Delete (remove / empty vote option) or similar key combos too. 
What are these bound to in other Apps like dude-al, lettuce, etc. ?
